databridge.camel-aas/src/main/java/org/eclipse/digitaltwin/basyx/databridge/aas/AASConsumer.java

```diff
@@ -0,0 +1,132 @@
+package org.eclipse.digitaltwin.basyx.databridge.aas;
+
+import org.apache.camel.AsyncCallback;
+import org.apache.camel.Exchange;
+import org.apache.camel.PollingConsumer;
+import org.apache.camel.Processor;
+import org.apache.camel.support.DefaultMessage;
+import org.apache.camel.support.ScheduledPollConsumer;
+import org.eclipse.basyx.submodel.metamodel.connected.ConnectedSubmodel;
+import org.eclipse.basyx.submodel.metamodel.connected.submodelelement.dataelement.ConnectedProperty;
+import org.eclipse.basyx.submodel.metamodel.facade.SubmodelElementMapCollectionConverter;
+import org.eclipse.basyx.vab.coder.json.serialization.DefaultTypeFactory;
+import org.eclipse.basyx.vab.coder.json.serialization.GSONTools;
+import org.eclipse.basyx.vab.modelprovider.VABElementProxy;
+import org.eclipse.basyx.vab.modelprovider.api.IModelProvider;
+import org.eclipse.basyx.vab.protocol.http.connector.HTTPConnectorFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author rana
+ *
+ */
+public class AASConsumer extends ScheduledPollConsumer implements PollingConsumer{
+	
```

Comment:
PollingConsumer is used instead of default consumer because we need to keep consuming data with time intervals, but the default can not create route by itself that's why it was used. 

![polling](https://github.com/eclipse-basyx/basyx-databridge/assets/129751857/c9bfde6d-81c2-488f-9dbd-efb2258412e6)